src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -540,13 +332,8 @@ private List<ModificationInfos> applyModification(NetworkStoreListener listener,
                 case GENERATOR_MODIFICATION:
                 case SUBSTATION_CREATION:
                 case TWO_WINDINGS_TRANSFORMER_CREATION:
-                    // Generic form
+                case BRANCH_STATUS_MODIFICATION:
```

Comment:
Remove the switch

Reply:
ok.
(I was hesitating to start the code cleaning)